frontend/src/lib/infer.ts

```diff
@@ -201,8 +201,6 @@ function argSigToJsonSchemaType(
 				delete oldS[prop]
 			}
 		}
-	} else if (oldS.format == 'date-time' && newS.format != 'date-time') {
-		delete oldS.format
```

Comment:
Instead of doing this, a condition should be added for languages that do not support native date-time. 
That would require quite a big refactor unfortunately